dependencies/README.md

```diff
@@ -0,0 +1,67 @@
+Static Linkage Checker
+======================
+
+Static Linkage Checker is a tool that finds [static linkage errors](
+../library-best-practices/glossary.md) on a classpath and reports the errors to the console.
+It scans the class files in a classpath provided as input for references
+to other classes.
+It then verifies each of the references to find linkage errors in the classpath.
+The tool also provides analysis on reachability to the errors from entry point
+classes through class usage graphs.
+
+### Use Cases
+
+There are two use cases for Static Linkage Checker:
+
+-  **For organizations** that provide multiple libraries developed by different teams,
+  the tool helps to ensure that there are no static linkage errors among the libraries and their
+  dependencies.
+
+- **For library/application developers** the tool finds static linkage
```

Comment:
I would put this one first. The other one is secondary; it's there only to make it less likely that a user using this tool will find errors that the library developer should have caught.